src/tracker.ts

```diff
@@ -127,7 +127,7 @@ function trackStructuredEvent(
  * @returns {Promise}
  */
 function trackPageViewEvent(
-  namespace: string,
+  namespace: string | null,
```

Comment:
I can think of one other alternative – to fetch the namespace of the default tracker from the native tracker in the TypeScript code and then use that namespace to track the events. We can change it to do that – it would require a new API to get the default tracker namespace in `RNSnowplowTrackerModule` and `RNSnowplowTracker` and exchanging two messages instead of one between the TS and native code. But it also seems like a good solution to me.

> we could create a tracker namespace type to use in all functions

I'm not sure I understand what you mean here?

Reply:
Sorry i was not clear. What i meant is that currently we have `trackPageViewEvent`, `trackStructuredEvent`, `trackSelfDescribingEvent` and `trackScreenViewEvent` accepting

```
namespace: string | null
```

and the rest of the functions (e.g. `trackTimingEvent`) accepting

```
namespace: string
```

Even though those functions cannot be called in the context of webviews, it may be a bit clearer to have the same type for this property. I was just suggesting a type definition to be applied for all. Not a blocking issue at all though, as it may not be worth it.

Reply:
Oh, I see, that's a good point, it's better to be consistent in all the internal track methods. I changed the others as well.